server/db/index.js

```diff
@@ -1,17 +1,15 @@
 const mongoose = require('mongoose');
 // const mongoUrl = 'mongodb://database/gallery';
-
-// mongoose.connect(mongoUrl, { server: { reconnectTries: Number.MAX_VALUE } })
-//   .then(() => {
-//     console.log('Successfully connected to MongoDb');
-//   })
-//   .catch((error) => console.log(error));
-
+const { Client } = require('@elastic/elasticsearch');
+const esclient = new Client({
+  node: 'http://localhost:9200',
```

Comment:
I would initiate a const at the top and use upper case letters for it's name.

Reply:
@galw you mean where `const mongoose = require('mongoose');` is?

Also, are you referring to the variable `esclient`? Decided to use lowercase `esclient` as that was the convention in official docs (it's probably not a class constructor).

Should I change it to `esClient` perhaps?